docs/Installation.md

```diff
@@ -1,265 +1,255 @@
+        ./database_build.py
+
+###  Step 7: Verify that the software-discovery-tool server is up and running
+ We now run the following commands to properly enable the config files of the software-discovery-tool server and then restart the apache server. 
+
+        sudo a2ensite z-000-default.conf
+        systemctl reload apache2
+        sudo a2ensite sdt.conf
+        systemctl reload apache2
+        sudo apachectl restart
+
+We can check if the server is up and running by going to following URL :
+
+```http://server_ip_or_fully_qualified_domain_name:port_number/sdt``` <br />
+
+(Alternatively, you can check with unittesting) <br />
+```cd software-discovery-tool/src/tests``` <br />
+
+If you run `pytest` as your logged user, it may give errors/warnings since you have given user `apache` ownership.
+
+		sudo -u apache pytest
+
+_**NOTE:**_ 
+
+* By default the port_number will be 80
+
+###  Step 8: (Optional) Custom configuration
+Following configuration settings can be managed in `/opt/software-discovery-tool/src/config/config.py`:
+
+        <software-discovery-tool_BASE> - Base location where software-discovery-tool is Installed/Cloned. Defaults to `/opt/software-discovery-tool/`
+
+        <DATA_FILE_LOCATION> - Location of folder containing all distribution specific JSON data
+        
+        <LOG_FILE_LOCATION> - Location of folder containing software-discovery-tool logs
+        
+        <enable_proxy_authentication> - Flag enabling/disabling proxy based network access
+        
+        <proxy_user> - Proxy server user name
+        
+        <proxy_password> - Proxy server password
+        
+        <proxy_server> - Proxy server IP/fully qualified domain name
+        
+        <proxy_port> - Proxy port number
+        
+        <DEBUG_LEVEL> - Set Debug levels for the application to log
+        
+        <server_host> - IP/fully qualified domain name of server where software-discovery-tool application will be deployed
+        
+        <server_port> - software-discovery-tool port on which application will be accessible to end users
+
+        <SUPPORTED_DISTROS> - Mapping of all the supported distros, new distros added need to be mapped here.
+
+        <MAX_RECORDS_TO_SEND> = Max number of records returned to the client. Defaults to 100
+
+        <CACHE_SIZE> - Number of searches to be cached. Default to 10
+
+_**NOTE:**_
+* In order to add new distribution support refer [here](Adding_new_distros.md)
+
+In case any of the parameters are updated, the server has to be restarted:
+
+#### Start/Restart Apache service
+
+        sudo apachectl restart
+
+###  Step 9: Start React (frontend) server
+
+#### Ensure Node.js and npm are installed
+
+    # Make sure you have Node.js and npm installed. You can download them from [here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+#### Change to the react-frontend directory
+
+    cd react-frontend
+
+#### Install the required npm packages
+
+    npm i

```

Comment:
This doesn't work as my local user due to /opt/software-discovery-tool permissions (owned by apache) and doesn't work when I run it with the apache user because the apache user doesn't have a /home directory.

Should this be run with sudo? That works for me.